openassessment/workflow/models.py

```diff
@@ -171,7 +171,7 @@ def start_workflow(cls, submission_uuid, step_names, on_init_params):
 
                 # If we auto-added a staff step, it is optional and should be marked complete immediately
                 if step.name == "staff" and staff_auto_added:
-                    step.assessment_completed_at=now()
+                    step.assessment_completed_at = now()
```

Comment:
Note to reviewers: A fair number of the changes in this file are due to my text editor flagging quality failures that were easy to address.